README.md

```diff
@@ -178,15 +178,71 @@ each milestone.
 
 ## JupyterHub
 
-### Next (0.4)
-
-- support user-provided inputs to Spawners via web form
-- allow users to have more than one server with different configurations
-
-### Next + 1 (0.5)
-
-- add services API for managing associated services (culling scripts, etc.)
-- provide one-way publishing/sharing mechanism
+### Not tied to releases
+
+JupyterHub constellation of repos may move to top-level jupyterhub org on GitHub.
```

Comment:
I missed this breakout, but this `may` should be `will` IMHO... otherwise the repos are wide dispersed in the main jup organization...